client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/use-steps.tsx

```diff
@@ -121,52 +124,60 @@ const useStepsData = ( { fromUrl }: StepsDataOptions ): StepsData => {
 		},
 		{
 			title: translate( 'Add your migration key' ),
-			content: (
-				<>
-					<p>
-						{ translate(
-							'Copy the key below. Head to the Migrate Guru settings on your source site, and paste it into the {{strong}}%(migrationKeyLabel)s{{/strong}} field.',
-							{
+			content:
+				'' === migrationKey ? (
+					<>
+						<p>{ translate( 'The key will be available here when your new site is ready.' ) }</p>
+						<p>
```

Comment:
Is this `<p>` necessary when there's already a `<div>`?

Reply:
Actually, I just noticed this warning in the console:

![Screenshot 2024-07-17 at 12 27 39 PM](https://github.com/user-attachments/assets/e4897eeb-426f-4828-a0d7-25a91d7e888e)

Reply:
Good catch! Fixed here: https://github.com/Automattic/wp-calypso/pull/92608/commits/ac46a0c6c8e9bf32843d12a928886e1172cb95b8